README.md

```diff
@@ -48,8 +48,10 @@ Here is a minimal policy example:
         "redshift-data:DescribeTable",
         "redshift-data:GetStatementResult",
         "redshift-data:DescribeStatement",
+        "redshift-data:ListStatements",
```

Comment:
I don't think this would be a breaking change since it adds a new feature. At least how it's currently implemented, ff async queries are enabled, then this would cause errors, but by default it should still go through the sync query flow which doesn't require these permissions. What would create the breaking change for this?